src/pybind/mgr/dashboard_v2/services/ceph_service.py

```diff
@@ -53,8 +133,96 @@ def get_service(cls, service_name, service_id):
 
     @classmethod
     def get_pool_list(cls, application=None):
-        osd_map = cls.mgr.get('osd_map')
-        if not application:
-            return osd_map['pools']
-        return [pool for pool in osd_map['pools']
-                if application in pool.get('application_metadata', {})]
+        return _get_pool_list(cls.mgr, application)
+
+
+class CephDaemonMixin(object):
```

Comment:
The name is misleading: It doesn't providing generic daemon services. in stead it is bound to the mirroring controller.